locales/templates/app.pot

```diff
@@ -667,26 +672,26 @@ msgctxt "screenshot_illustration"
 msgid "Capturing screenshot of entire page…"
 msgstr ""
 
-#. Available action in Snackbar that gives users a chance to view the captured
-#. screenshot     right after it is successfully captured.
+#. Available action shown along with screenshot_saved that gives users a
+#. chance to view the captured screenshot.
```

Comment:
Actually, I feel like it would make more sense if you put something like "Available action in a temporary notification (message?) that gives users a chance to view the captured screenshot."
Same for the comments below.
Or something along those lines, if that makes sense

Reply:
I meant, "same as for the other localization notes below, when they call out screenshot_saved"

Reply:
Sure. I think I'll pick message here since the term Notification is quite already occupied by the system Notifications on mobile and I'd like to avoid misleading here.

Reply:
Good point. That definitely WFM

Reply:
Turns out I found another way which I like even more to cover all of what we've discussed. Hope that's working! (So proud of myself to be able to come up with this at this time, probably I'm still recovering from the All-hands jet lag. Thanks to all-hands 😝 .)